config/mainnet.go

```diff
@@ -136,11 +136,12 @@ func MainnetConfig() Config {
 		FETCH:    fetch.DefaultConfig(),
 		LOGGING:  logging,
 		Sync: syncer.Config{
-			Interval:         time.Minute,
-			EpochEndFraction: 0.8,
-			MaxStaleDuration: time.Hour,
-			Standalone:       false,
-			GossipDuration:   50 * time.Second,
+			Interval:           time.Minute,
+			EpochEndFraction:   0.8,
+			MaxStaleDuration:   time.Hour,
+			Standalone:         false,
+			GossipDuration:     50 * time.Second,
+			OutOfSyncThreshold: 36, // 3h
```

Comment:
I think the name should indicate that these are layers, maybe `LayerOutOfSyncThreshold`? Took me a bit to figure out how 36 means 3 h 😅